src/components/Section/Foreign/Business/Employment.jsx

```diff
@@ -42,14 +42,14 @@ export default class Employment extends SubsectionElement {
   }
 
   summary(item, index) {
-    const obj = item || {}
-    const date = DateSummary(item.Date)
-    const job = `${(obj.Description || {}).value || ''}`.trim() || ''
+    const o = (item || {}).Item
```

Comment:
Based on the issue we resolved above in `Foreign/Activities/Support.jsx`, we may want to add the ` || {}` back in here just in case, thoughts?

Reply:
Yeah that makes sense 👍